doc/Roadmap.md

```diff
@@ -24,43 +24,51 @@ Please contact us and tell us about what features and improvements do you see as
 * Experimental pipelining
 * Instant Stream Resumption
 * Bind 2.0 (experimental)
-* Basic one-to-one VoIP (NAT traversal and media relaying with ICE server)
+* Basic one-to-one VoIP (NAT traversal and media relaying with ICE/STUN/TURN server)
 * Contributions to ecosystem
 
 ## MongooseIM 3.0.x: chatbots and IoT (2017S2)
 
 * Conversational model:
-    * In-chat forms, for bots and humans: new ProtoXEP, sample implementations on server and clients
+    * In-chat forms (quick replies), for bots and humans: new ProtoXEP, sample implementations on server and clients
     * Ephemeral messages?
     * Foldable content?
 * IoT:
     * things provisioning?
     * OCF certification?
+    * Embedded client?
+    * Cloud/fog focus?
 * SASL 2.0?
-* Stories?
 * [XEP-0376: Pubsub Account Management](https://xmpp.org/extensions/xep-0376.html)?
 * Contributions? [XEP-xxxx: Explicit Message Encryption](https://xmpp.org/extensions/inbox/eme.html)?
 
-## MongooseIM 3.1.x: chatbots and IoT (2018)
+## MongooseIM 3.1.x: chatbots and IoT continuity, social (2018)
 
 * [XEP-0156: Discovering Alternative XMPP Connection Methods](http://xmpp.org/extensions/xep-0156.html)?
 * [XEP-0379: Pre-Authenticated Roster Subscription](https://xmpp.org/extensions/xep-0379.html)?
 * Rich content
+* Social networking improvements
+* Stories?
 * Contributions?
 
-## MongooseIM platform (2017)
+## Current state of the MongooseIM platform
 
 This is not the MongooseIM XMPP/REST messaging server, all these components are part of the MongooseIM platform.
```

Comment:
The following components are a part of the MongooseIM platform, not the XMPP/REST messaging server.